src/bundle/DependencyInjection/ContainerBuilder.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Bundle\CompatibilityLayer\DependencyInjection;
+
+use Symfony\Component\DependencyInjection\ContainerBuilder as SymfonyContainerBuilder;
+use Symfony\Component\DependencyInjection\Extension\ExtensionInterface;
+
+/**
+ * @internal
+ */
+class ContainerBuilder extends SymfonyContainerBuilder
```

Comment:
Why it isn't `final` ?

Reply:
> Why it isn't `final`?

Hmm, maybe it should until it's needed to be extended. Fixed via 6c5a203.